ui/src/app/machines/views/MachineList/MachineList.scss

```diff
@@ -1,5 +1,7 @@
+$grouped-machines-indentation: 2.5rem;
+
 .machine-list--grouped .machine-list__machine td:first-child {
-  padding-left: 2rem;
+  padding-left: $grouped-machines-indentation;
```

Comment:
Ideally this should be a sum of whatever spaces make up this value - e.g. checkbox width + 2*$sph-inner or whatever it is. SO if any of these changes, this would stay in sync.

Reply:
Gotcha. I'll make the change and merge in a moment.